src/include/cluster/wizards.rb

```diff
@@ -101,13 +102,18 @@ def FirstRunSequence
       sequence = {
         "ws_start"      => "communication",
         "communication" => {
+          :next  => "corosyncqdevice",
+          :back  => "communication",
```

Comment:
this looks strange, having back to itself. It back needed here at all? maybe it should be just disabled.

Reply:
“Back” button still appear when removing line 106. So leave the code without change.